### ISSUE TITLE
Internal tag of parent not applied for nested struct (Flutter)

### ISSUE THREAD

**Issue by Hsilgos** [heremaps/gluecodium#1602]
For Flutter, the nested struct of an internal struct is exposed.

```
package sdk.core

@Internal
struct OuterStruct {
    struct InnerStruct {
        a: Int = 1
    }

    inner: InnerStruct = {}
}
```

Leads to

```
export 'src/sdk/core/outer_struct.dart' show OuterStruct_InnerStruct;
```
Expected: inter struct not exposed 

Current workaround:
Add @Internal tag to nested struct.

**Comment by hliberacki** [comment]
Hi @Hsilgos,

As we discussed in #1601, I will try implementing it.

**Comment by pwrobeldev** [comment]
The fix has been implemented in: https://github.com/heremaps/gluecodium/pull/1606

**Comment by pwrobeldev** [comment]
The fix was released in 13.9.6. Closing the issue.
